oak-store-spi/src/test/java/org/apache/jackrabbit/oak/json/JsonSerializerTest.java

```diff
@@ -63,13 +126,59 @@ public void childOrder() throws Exception{
             }
 
         } while (reader.matches(','));
+        Assert.assertTrue(customLogs.getLogs().stream().filter(n -> n.contains("Json is not getting flushed in chunks:")).findFirst().isPresent());
```

Comment:
this could be simplified
```suggestion
        Assert.assertTrue(customLogs.getLogs().stream().anyMatch(n -> n.contains("Json is not getting flushed in chunks:")));
```